SPEC.md

```diff
@@ -266,9 +266,11 @@ public class Car
 }
 ```
 
+If the specified property name conflicts with a field name of the same class, the property will get precedence over the field during serialiation and deserialization.
+
 ### Methods
 
-By default, C# methods are ignored, however, provided the “get” signature is a no-argument method and the “set” signature is a one-argument `void` method, methods can be use to get and set properties. The Ion property name must be specified and will not be inferred from the method name.
+By default, C# methods are ignored, however, provided the “get” signature is a no-argument method and the “set” signature is a one-argument `void` method, methods can be use to get and set properties. The Ion property name must be specified and will not be inferred from the method name. In the event of a naming conflict between the specified annotated Ion property name and a property or field name of the class, the annotated Ion property name will get precedence over the class's property or field name during serialization and deserialization.
```

Comment:
Can you provide an example here?

Reply:
Funnily enough the sample snippet below is such an example.

Reply:
Not quite though, right? Because color is private, you would need it to be public or you would need a public annotated Property?

Reply:
The method serialization logic still applies to this example even though the field is private. The only change needed to make this example have a naming conflict is to annotate the color field with `[IonField]` (because fields are ignored by default).